net/uradvd/Makefile

```diff
@@ -0,0 +1,42 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=uradvd
+PKG_VERSION:=1
+
+PKG_MAINTAINER:=Moritz Warning <moritzwarning@web.de>
+PKG_LICENSE:=MIT
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/uradvd
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Advertise an IPv6 prefix/route
+  DEPENDS:=+librt
```

Comment:
I'm not sure you need explicit depency on librt (it added automatically)

Reply:
Yeah, the Makefile is ancient and this can be removed - it might have been necessary with uClibc in the past.